contracts/abstracts/ListingManager.sol

```diff
@@ -40,4 +40,19 @@ abstract contract ListingManager is Controlable {
     // To-Do: Send sale amount minus fees to seller
 
   }
+<<<<<<< Updated upstream
+=======
```

Comment:
These two line and the one below the function isSold, should not be there

Reply:
j'ai dû faire une mauvaise manipulation de branch, du coup, il a mélangé les modifications que j'ai faites d'une autre branch :(